techniques/aria/ARIA11.html

```diff
@@ -134,13 +137,13 @@ <h2>Related Techniques</h2>
    <section id="resources"><h2>Resources</h2>   
       <ul>
          <li>
-            <a href="https://www.w3.org/TR/wai-aria-practices/">WAI-ARIA Authoring Practices</a> 
+            <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#aria_landmark">WAI-ARIA Authoring Practices 1.1, Landmark Regions</a> 
```

Comment:
the apg isn't going on point releases anymore.  here is the latest link:

```suggestion
            <a href="https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/">WAI-ARIA Authoring Practices, Landmark Regions</a> 
```